contollers/todo.controller.js

```diff
@@ -18,7 +19,11 @@ class TodoController {
         description: req.body.description,
         userId: req.user.id,
       });
-
+      console.log(todo.id);
+      if (todo.userId)
+        await List.findByIdAndUpdate(req.body.listId, {
```

Comment:
I dont think its the place to push to a list, and you're pushing to a list without making sure that `listId` exists in `req.body`, although `listId` is not required here.. 
I think that the right place to add todos to a list, is in the lists routes, not here.
I have a plan for this, I really suggest that you postpone this for next phase